debian/cloudstack-agent.postinst

```diff
@@ -46,6 +46,11 @@ case "$1" in
             fi
         fi
 
+        # Create symbolic link for python
+        if [ ! -f "/usr/bin/python" ]; then
+            ln -s /usr/bin/python2 /usr/bin/python
```

Comment:
Hi @weizhouapache - my colleagues @shwstppr @davidjumani are working to support python2 to python3 in #4068, so all scripts that run on management server and kvm host towards 4.15/milestone. So this change is not needed, and may break things (we probably don't want misleading symlinks that could mean python2 or python3). cc @wido @GabrielBrascher

Reply:
@rhtyd @wido 
I do not know when cloudstack 4.15 will be released (with python3).
this can be regarded as a patch for cloudstack 4.14.
it works with same mysql version, same scripts (some written in python2, some written in python3), etc